CONTRIBUTING.adoc

```diff
@@ -122,6 +191,7 @@ and people may not be notified if you just push commits.
 
```

Comment:
I'd add a short list of commands to execute before pushing (e.g. stylish, nix regenerate etc.) here.

Reply:
Added, thanks!